vb-be/src/modules/socket/socket.gateway.ts

```diff
@@ -7,29 +6,42 @@ import {
   ConnectedSocket,
 } from '@nestjs/websockets';
 import { Server, Socket } from 'socket.io';
-import { WsGuard } from './socket.middleware';
-import { SocketService } from './socket.service';
+
+import { socketError, ErrorText } from 'src/util/error';
+import { HttpStatus } from 'src/util/http';
+
+import { RoomService } from '@modules/room/room.service';
+import { ISocketCreateRoomData } from '@modules/socket/socket.constants';
 
 @WebSocketGateway({ cors: true })
 export class EventsGateway {
   @WebSocketServer()
   server: Server;
 
-  constructor(private readonly socketService: SocketService) {}
+  constructor(private readonly roomService: RoomService) {}
 
-  // @UseGuards(WsGuard)
-  @SubscribeMessage('create-room')
-  async joinRoom(@MessageBody() data: any, @ConnectedSocket() client: Socket) {}
+  // // @UseGuards(WsGuard)
```

Comment:
Did you mean to leave it all commented out?

Reply:
yeah wasn't using this file